src/Pages/Authentication/Signup.jsx

```diff
@@ -0,0 +1,125 @@
+import "../../css/main.css";
+import "./authentication.css";
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { useAuth } from "../../Context";
+
+
+export const SignUp = () => {
+    const { authState, signUp } = useAuth();
+
+    const [userDetails, setUserDetails] = useState({
+        firstName: "",
+        lastName: "",
+        email: "",
+        password: "",
+        confirmPassword: "",
+        terms: false,
+    })
+
+    const [error,setError] = useState({isError: false, text: ""})
+
+    const [showPasswordToggle, setShowPasswordToggle] = useState(true)
+
+    useEffect(() => {
+        const timeoutID = setTimeout(() => {
+            setError({...error, isError: false});
+        }, 3000)
+
+        return () => clearTimeout(timeoutID);
+    },[error])
+
+    const submitHandler = (e) => {
+
+        e.preventDefault();
+
+        const passwordValidation = /^(?=.*\d)(?=.*[a-z]).{5,10}$/;
```

Comment:
in regex password its not accepting characters more than 10 ....because of {5,10} make it {5,} to accept minimum value
i.e.    `const passwordValidation = /^(?=.*\d)(?=.*[a-z]).{5,}$/;`

Reply:
Yes will change it. Thanks for pointing this out.